Tools/LyTestTools/tests/unit/test_file_system.py

```diff
@@ -208,20 +208,6 @@ def test_Unzip_ReleaseBuild_ReturnsCorrectPath(self, mock_join, mock_check_free)
 
         self.assertEqual(path, expected_path)
 
-    @mock.patch('os.path.exists')
```

Comment:
These zip/unzip utilities have not been used by tests in ages. Paths resolve differently between Windows and non-windows, so the mock-assert is invalid outside of Windows. It was easier to remove these low-value tests.